agent-info/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
```

Comment:
Why do we need this to be in a dedicated crate?

Reply:
I figured several other crates might need it in the future and it seemed a bit to complex to be in ddcommon.

Reply:
Unless I'm misunderstanding. the complexity exists solely because of our specific use-case and implementation of client-side stats. 

I can see the value in having the info endpoint's schema along with some possible constants for routes, etc in `ddcommon` (or somewhere else appropriate for common code) to reduce boilerplate around making http requests.  

I can also see value in a crate that offers some sort of generic event-driven solution for polling and notifying consumers of HTTP endpoint changes (i'm surprised this doesn't already exist). 

But this PR feels very un-generic. I'm not suggesting we have to come up with a different solution, but I think it should just live in data-pipeline, at least for now.

Reply:
So something like, moving the `fetch_info` function to ddcommon maybe behind a feature, and the `AgentInfoFetcher` into `data_pipeline` ?

Reply:
I am ok with all of it living in `data-pipeline` for now since this is the only current use-case I'm aware of and refactoring later wouldn't be a big enough of an effort to violate [the rule of threes](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)).

But, if you did want to put something in ddcommon I'd suggest just the schema.